.doks/doks-cluster/databases.tf

```diff
@@ -0,0 +1,213 @@
+######################
+# cluster-dev
```

Comment:
Can-t we just reuse this template 3 times with a variable that recieves the name of the DB?

Reply:
We probably can. It did not occur to me 👍 
I did that for the dbs but not for the clusters themselves. Changing it now.

Reply:
Although that will force the replacement (destruction and creation) of the environments. Meaning, re-importing the dbs :-(